SpatialGDK/Source/SpatialGDK/Private/SimulatedPlayers/SimulatedPlayer.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SimulatedPlayer.h"
+
+#include "Misc/Parse.h"
+#include "Misc/CommandLine.h"
+#include "SpatialConstants.h"
+
+DEFINE_LOG_CATEGORY_STATIC(LogSimulatedPlayer, Log, All);
+
+bool USimulatedPlayer::IsSimulatedPlayer()
+{
+#if WITH_EDITOR
```

Comment:
Why do we use the preprocessor for this rather than a check?

Reply:
That's gone now 👍